src/rules/__tests__/prefer-jest-globals.test.ts

```diff
@@ -0,0 +1,36 @@
+import { TSESLint } from '@typescript-eslint/utils';
+import dedent from 'dedent';
+import rule from '../prefer-jest-globals';
+import { espreeParser } from './test-utils';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: espreeParser,
+  parserOptions: {
+    ecmaVersion: 2015,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('prefer-jest-globals.test', rule, {
```

Comment:
we need more tests than this - even though this should just be a thin wrapper for `parseJestFnCall` which already has a lot of tests, we should stick a few here anyway, including checking a few different calls with chains and aliases (both via `import` and doing e.g. `const context = describe`, which it's fair _not_ to report on `context`)

Reply:
Added new alias units tests